src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    guarantees data reliability. Using a value less than the canonical one
+    might lead to unexpected results, including a split-brain.
+
+    `replication.synchro_quorum` is not used on replicas. If the master fails,
+    the pending synchronous transactions will be kept waiting on the replicas
+    until a new master is elected.
+]])
+
+M['replication.synchro_timeout'] = format_text([[
+    For synchronous replication only. Specify how many seconds to wait for a
+    synchronous transaction quorum replication until it is declared failed and
+    is rolled back.
+
+    It is not used on replicas, so if the master fails, the pending synchronous
+    transactions will be kept waiting on the replicas until a new master is
+    elected.
+]])
+
+M['replication.threads'] = format_text([[
+    The number of threads spawned to decode the incoming replication data.
+
+    In most cases, one thread is enough for all incoming data. Possible values
+    range from 1 to 1000. If there are multiple replication threads, connections
+    to serve are distributed evenly between the threads.
+]])
+
+M['replication.timeout'] = format_text([[
+    A time interval (in seconds) used by a master to send heartbeat requests to
+    a replica when there are no updates to send to this replica. For each
+    request, a replica should return a heartbeat acknowledgment.
+
+    If a master or replica gets no heartbeat message for
+    `4 * replication.timeout` seconds, a connection is dropped and a replica
+    tries to reconnect to the master.
+]])
+
+-- }}} replication configuration
+
+-- {{{ roles configuration
+
+M['roles'] = format_text([[
+    Specify the roles of an instance. To specify a role's configuration,
+    use the roles_cfg option.
+]])
+
+M['roles.*'] = 'Role name.'
+
+-- }}} roles configuration
+
+-- {{{ roles_cfg configuration
+
+M['roles_cfg'] = format_text([[
+    Specify a role's configuration. This option accepts a role name as the
+    key and a role's configuration as the value. To specify the roles of an
+    instance, use the roles option.
+]])
+
+M['roles_cfg.*'] = 'Role name.'
+
+-- }}} roles_cfg configuration
+
+-- {{{ security configuration
+
+M['security'] = format_text([[
+    This section defines configuration parameters related to various
+    security settings.
+]])
+
+M['security.auth_delay'] = format_text([[
+    Specify a period of time (in seconds) that a specific user should wait for
+    the next attempt after failed authentication.
+
+    The `security.auth_retries` option lets a client try to authenticate the
+    specified number of times before `security.auth_delay` is enforced.
+
+    In the configuration below, Tarantool lets a client try to authenticate
+    with the same username three times. At the fourth attempt, the
+    authentication delay configured with `security.auth_delay` is enforced.
+    This means that a client should wait 10 seconds after the first failed
+    attempt.
```

Comment:
> In the configuration below <...>

It refers to something that doesn't exist here. There is nothing below.

Reply:
Dropped.